@here/harp-mapview/lib/MapView.ts

```diff
@@ -1390,6 +1392,16 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_lookAtDistance;
     }
 
+    /**
+     * The maximum visibility range (in meters) that may be achievied with current camera setttings.
```

Comment:
I think the comment `(in meters)` is correct only at the equator, or for globe projection, for any planar projection, I think this is incorrect, because the values distort towards the equator. Maybe world units, or projected units is best?

Reply:
True.